Megaparsec/Lisp.lean

```diff
@@ -76,7 +78,7 @@ structure LispLinearParsers where
   stringP :=
     s.label "string" $ do
     let (str : String) ←
-      between (c.char '"') (c.char '"') $
+      Seq.between (c.char '"') (c.char '"') $
```

Comment:
Wait what, there was `Seq.between` this whole time or YatimaStdLib somehow hooked into the namespace?